internal/cli/atlas/deployments/logs_test.go

```diff
@@ -85,45 +83,30 @@ func TestLogs_RunLocal(t *testing.T) {
 func TestLogs_RunAtlas(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	ctx := context.Background()
-	buf := new(bytes.Buffer)
 	atlasDeployment := "localDeployment1"
 	mockStore := mocks.NewMockLogsDownloader(ctrl)
 	deploymentTest := fixture.NewMockAtlasDeploymentOpts(ctrl, atlasDeployment)
 
 	downloadOpts := &DownloadOpts{
-		OutputOpts:     cli.OutputOpts{OutWriter: buf},
-		GlobalOpts:     cli.GlobalOpts{ProjectID: "ProjectID"},
-		DownloaderOpts: cli.DownloaderOpts{},
+		GlobalOpts: cli.GlobalOpts{ProjectID: "ProjectID"},
+		DownloaderOpts: cli.DownloaderOpts{
+			Out: "out",
+		},
 		DeploymentOpts: *deploymentTest.Opts,
 		downloadStore:  mockStore,
 		host:           "test",
 		name:           "mongodb.gz",
 	}
 
 	downloadOpts.Fs = afero.NewMemMapFs()
-	downloadOpts.Out = downloadOpts.name
 	deploymentTest.CommonAtlasMocks(downloadOpts.ProjectID)
 
+	b, _ := base64.RawStdEncoding.DecodeString("H4sIAAAAAAAA/8pIzcnJVyjPL8pJAQQAAP//hRFKDQsAAAA") // "hello world" gzipped
```

Comment:
Nice!

Reply:
yeap, that is to test the decompression code